ckanext/push_errors/tests/test_push_menssage.py

```diff
@@ -0,0 +1,43 @@
+from unittest.mock import patch, MagicMock
+from ckanext.push_errors.logging import push_message
+
+
+@patch('ckanext.push_errors.logging.toolkit')
+@patch('ckanext.push_errors.logging.requests.post')
+def test_push_message_success(mock_post, mock_toolkit):
+    # Configuración de mocks
+    mock_toolkit.config.get.side_effect = lambda key, default=None: {
+        'ckanext.push_errors.url': 'https://fake-url.org',
+        'ckan.site_url': 'https://mysite.org',
+        'ckanext.push_errors.method': 'POST',
+        'ckanext.push_errors.headers': '{}',
+        'ckanext.push_errors.data': '{}',
+        'ckanext.push_errors.message_title': 'Test Message {site_url} - {now}'
+    }.get(key, default)
```

Comment:
Use `@pytest.mark.ckan_config` here in all tests instead mocking `config.get`

Reply:
https://github.com/unckan/ckanext-push-errors/pull/12/commits/48b4af99ee0d7d5e9174027b842b25513040999b